doc/dependency_decisions.yml

```diff
@@ -193,27 +193,27 @@
 - - :license
   - "@viamrobotics/rpc"
   - MIT
-  - :who: 
+  - :who:
     :why: 'TODO(RSDK-583): why is license finder unable to detect this license automatically?'
     :license_links: https://github.com/viamrobotics/goutils/blob/6684974ace14045e04d479777852602817e03313/LICENSE
     :versions: []
     :when: 2022-08-31 20:08:14.585047144 Z
 - - :approve
   - github.com/viamrobotics/evdev
-  - :who: 
+  - :who:
     :why: license language is compatible with AGPL
     :license_links: https://github.com/viamrobotics/evdev/blob/5a415229fba9557cca9bfedc7ba89cb48f28fab0/LICENSE
     :versions: []
     :when: 2022-08-31 20:13:08.147340927 Z
 - - :permit
   - GPLv2+
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2022-09-16 15:13:54.129990412 Z
 - - :approve
   - github.com/opencensus-integrations/gomongowrapper
-  - :who: 
-    :why: 
+  - :who:
+    :why:
```

Comment:
Do we need a new entry for `semver`?

Reply:
Semver uses an MIT license which has a blanket approval.